todo/views.py

```diff
@@ -17,8 +17,35 @@ def createpage(request):
         todo.save()
         return redirect('home')
     return render(request,'create.html')
+
+def delete(request, todo_id):
+    '''delete a todo'''
+    todo=Todo.objects.get(id=todo_id)
```

Comment:
User should be able to delete only his todo. If user enter a todo which does not exists or does not belongs to him, he must get a validation error.

Reply:
Validation add krta hun men